tests/generators/ssz_generic/ssz_container.py

```diff
@@ -118,3 +118,20 @@ def invalid_cases():
                                   offset_index=offset_index,
                                   change=lambda x: 0
                               ))
+                        if index == 0:
+                            yield f'{name}_{mode.to_name()}_first offset_{offset_index}_minus_one', \
```

Comment:
let's do "first offset" or just the offset index, and also i think this string should have no spaces

```suggestion
                            yield f'{name}_{mode.to_name()}_offset_{offset_index}_minus_one', \
```